core/src/main/java/org/bitcoinj/crypto/bls/BLSDeterministicKey.java

```diff
@@ -94,12 +94,10 @@ public BLSDeterministicKey(ImmutableList<ChildNumber> childNumberPath,
             bytes = serialize(null, false, Script.ScriptType.P2PKH);
             extendedPrivateKey = ExtendedPrivateKey.fromBytes(bytes);
             extendedPublicKey = extendedPrivateKey.getExtendedPublicKey();
-            this.parentFingerprint = (parent != null) ? (int)extendedPrivateKey.getParentFingerprint() : 0;
         } else {
             bytes = serialize(null, true, Script.ScriptType.P2PKH);
             extendedPrivateKey = null;
             extendedPublicKey = ExtendedPublicKey.fromBytes(bytes);
-            this.parentFingerprint = (parent != null) ? (int)extendedPublicKey.getParentFingerprint() : 0;
```

Comment:
This resolved an outstanding issue from the previous PR: https://github.com/dashpay/dashj/pull/199#discussion_r1133025331

These lines are removed as `parentFingerprint` is assigned in line 91.